lib/_http_client.js

```diff
@@ -311,11 +327,10 @@ ClientRequest.prototype._implicitHeader = function _implicitHeader() {
                     this[kOutHeaders]);
 };
 
-ClientRequest.prototype.abort = function abort() {
-  if (!this.aborted) {
-    process.nextTick(emitAbortNT.bind(this));
-  }
+ClientRequest.prototype.destroy = destroyImpl.destroy;
+ClientRequest.prototype._destroy = function destroy(err, cb) {
   this.aborted = true;
+  process.nextTick(emitAbortNT.bind(this));
```

Comment:
We can avoid the unnecessary `bind()` here by instead doing `process.nextTick(emitAbortNT, this);` and modifying `emitAbortNT` appropriately.